pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -655,6 +703,40 @@ func (computeCfg *computeServiceConfig) getATGroupView(nepheControllerATSGNameTo
 	return enforcedSecurityCloudView, nil
 }
 
+// isValidAgSg finds if an ASG is in managed VNET, and it is a valid nephe created AddressGroup SG.
+func (computeCfg *computeServiceConfig) isValidAgSg(asgIDLowercase string,
```

Comment:
same here.. also see if you can move functions together. isValidAG and AT onces . back to back..

Reply:
I didn't want to club them together as both the functions have different signatures.

Reply:
Re-checked, new functions are placed right, re-arranged an existing function to club all AT SG related functions together, similarly AG.